src/Utility/ExprFunctions.h

```diff
@@ -206,7 +206,7 @@ class EvalMultiFunction : public ParentFunc, public EvalFunctions<Scalar>
 
 public:
   //! \brief The constructor parses the expression string for each component.
-  EvalMultiFunction(const std::string& functions,
+  explicit EvalMultiFunction(const std::string& functions,
                     const std::string& variables = "",
                     const Real epsX = 1e-8,
                     const Real epsT = 1e-12)
```

Comment:
The indent for the continuation lines is now wrong

Reply:
fixed.